.eslintrc.js

```diff
@@ -8,7 +8,7 @@ const NEVER = 'never';
 
 module.exports = {
     'parserOptions': {
-        ecmaVersion: 2017
+        ecmaVersion: 2019
```

Comment:
Is it possible to go to airbnb here?

Reply:
We have task to rewrite this component, from scratch in such task yes, in this task no.